src/components/common/ProgressBar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Progress } from 'antd';
+
+export default function ProgressBar() {
+  // make the ProgressBar dynamic with props and plug it into the Progress component twice
+  return (
+    <div className="progress-bars">
+      <div className="progress-bar progress-bar-1">
```

Comment:
Does this specific div needs a classname 'progress-bar progress-bar-1'? Can it be just 'progress-bar 1'? I just feel like it would be hard to remember the name of it, if, let's say you want to style this div in index.less... It would be:
```
.progress-bars .progress-bar progress-bar-1{

}
```
Like, too many progress-bars :)